docs/source/developers.md

```diff
@@ -7,23 +7,39 @@ On Windows system, we recommend using WSL, the Windows Subsystem for Linux, beca
 Inside VSCode, please install the extensions that are recommended for this project - they are available in `.vscode/extensions.json` in the
 repository root.
 
+## Opening the repository
+
+Once you have the repository on your computer, you can open either all projects
+at once or individual projects separately in VSCode.
+
+* To open all projects at once, use VSCode's "Open Workspace from File", and
+  select `himl-projects.code-workspace`.
+* To open individual projects, use VSCode's "Open Folder", and select one of the
+  folders `hi-ml-azure`, `hi-ml`, or `hi-ml-histopathology`
+
 ## Creating a Conda environment
 
-To create a separate Conda environment with all packages that `hi-ml` requires for running and testing,
-use the provided `environment.yml` file. You can create a Conda environment called `himl` from that via either
+Different projects in this repository use different Conda environments:
+
+* The `himl` Conda environment should be used for work on the `hi-ml` and `hi-ml-azure` projects.
+* The `HimlHisto` Conda environment should be used for work on `hi-ml-histopathology`.
+
+Please select the right Python interpreter for your project (or all projects if using the `himl-projects` workspace)
+inside VSCode, by choosing "Python: Select Interpreter" from the command palette (Ctrl-Shift-P on VSCode for Windows)
+
+To create the Conda environment `himl`, please use either
 
 ```shell script
 conda env create --file environment.yml
```

Comment:
Should this be updated to `hi-ml/environment.yml`?

Reply:
yes, thanks!